packages/nc-gui-v2/utils/projectCreateUtils.ts

```diff
@@ -209,4 +144,19 @@ export const getDefaultConnectionConfig = (client: ClientType): ProjectCreateFor
   }
 }
 
-export const sslUsage = ['No', 'Preferred', 'Required', 'Required-CA', 'Required-IDENTITY']
+enum sslUsage {
```

Comment:
Enums should be Capitalized 😅 -> `SSLUsage` or `SslUsage`
So it's obvious they're not regular variables.